lib/utils.js

```diff
@@ -90,4 +90,18 @@ async function adjustWDAAttachmentsPermissions (perms) {
   }
 }
 
-export { detectUdid, getAndCheckXcodeVersion, getAndCheckIosSdkVersion, killAppUsingAppName, adjustWDAAttachmentsPermissions };
+async function generateXcodeConfigFile (orgId, signingId) {
+  log.debug(`Generating xcode config file for orgId ${orgId} and signingId ` +
```

Comment:
Can you put the `orgId` and `signingId` in single quotes? It makes it easier to see at a glance if something is wrong with the values (like they accidentally put a space in it or something).